examples/hello_world/test_driver/smoke_web_engine.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter_driver/driver_extension.dart';
```

Comment:
This and other new files need a license header.

Reply:
done